server/server.js

```diff
@@ -72,30 +52,20 @@ module.exports = function server(options = {}) {
       );
   });
 
-  app.post("/import", checkConfiguration, ({ agent }, res) => {
-    agent.async("startImport");
+  app.post("/import", checkConfiguration, (req, res) => {
+    req.hull.enqueue("startImport");
     res.json({ status: "scheduled" });
   });
 
-  app.post("/sync", checkConfiguration, ({ agent }, res) => {
+  app.post("/sync", checkConfiguration, ({ agent }, req, res) => {
```

Comment:
The signature of the handler callback is different here than for `/import` route.
Are you sure that the `/sync` works? you can trigger and test it using some rest API client (e.g. curl), but don't forget about the credentials.